### PR TITLE
Binomial_Distribution_(User-Defined)

### PR DESCRIPTION
**Pull request by Dekode1859**
Find the mean and variance of a binomial distribution of number of coin flips (X), probability of heads (Y) and number of samples (N) in it with optional seed value (S) all variables are user defined.

### CODE REVIEW

**Comment by dikshantrajput** [comment]
@Dekode1859 Please add your name to the [contributors.html](https://github.com/maheshjainckd/Hacktoberfest2022-for-everyone/blob/main/contributors.html) file

**Comment by Dekode1859** [comment]
@dikshantrajput Added my name to contributors.html

File: contributors.html [hunk 1]
```diff
@@ -115,6 +115,10 @@ <h1>
         <li>
             Khushi Marothi
         </li>
+        
+        <<li>
```
Comment (dikshantrajput) [inline contributors.html:119]: It should be <li> not <<li> and </li> not </li>>. Please fix it

**Review by dikshantrajput — COMMENTED** [review]
Do the changes requestes. Thanks

**Review by dikshantrajput — APPROVED** [review]

**Pull request merged**